libs/schematic/api-description/src/components/schemas/ValidationRule.yaml

```diff
@@ -0,0 +1,11 @@
+type: object
+description: A validation rule.
+properties:
+  name:
+    type: string
+    description: The name of the rule.
```

Comment:
Kind of small but could you update the description (or the higher level description) to include that the rule arguments are included as well? In schematic the rule and the arguments are treated as separate parts in a way and I think it would prevent confusion in the future to be clear about what's included.